riptide_mapping/riptide_mapping2/dummydetections.py

```
@@ -63,7 +52,7 @@ def __init__(self):
         self.pubs        = [ ]
         self.srv = self.create_service(SetBool, 'set_camera_is_dfc', self.setActiveCameraCb)
 
-        for object in objects:
+        for object in self.objects:
```

Ruff (0.15.17)

[error] Variable `object` is shadowing a Python builtin

(A001)